content/applications/hr/appraisals/skills_evolution.rst

```diff
@@ -0,0 +1,149 @@
+================
+Skills evolution
+================
+
+In Odoo's **Appraisals** app, it is possible to view employee's skills as they progress over time in
+the :ref:`Skills Evolution <appraisals/identify-skill-evolution>` report.
```

Comment:
```suggestion
the :ref:`Skills Evolution <appraisals/identify-skill-evolution>` report, also known as the *Appraisal Skills Report*.
```
this seems like a necessary addition as the title of the Reporting page differs from the drop-down Reporting option in this case. This should, hopefully, avoid any potential confusion from the reader as the documentation continues.